warp10/src/main/sh/warp10.sh

```diff
@@ -257,7 +249,17 @@ backend = leveldb" >>"${WARP10_CONFIG_DIR}/99-init.conf"
   chmod 700 "${LEVELDB_HOME}"
 
   ${JAVACMD} -cp "${WARP10_JAR}" io.warp10.standalone.WarpInit "${LEVELDB_HOME}" >>"${WARP10_HOME}/logs/warp10.log" 2>&1
+}
+
+leveldbConf() {
+  echo "Initializing Warp 10 standalone configuration"
+  init
 
+  echo "
+backend = leveldb" >>"${WARP10_CONFIG_DIR}/99-init.conf"
+  mv "${WARP10_CONFIG_DIR}/10-fdb.conf" "${WARP10_CONFIG_DIR}/10-fdb.conf.DISABLE"
```

Comment:
We should suffix the .DISABLE with a timestamp `YYYYMMDDTHHMMSS` to avoid overwriting an existing `.DISABLE` file.